modules/custom/az_core/az_core.module

```diff
@@ -266,7 +266,7 @@ function az_core_canonical_url(array $data) {
   $node = $data['node'];
   /** @var \Drupal\Core\Url $url */
   $url = $node->toUrl();
-  $uri = $url->toUriString();
+  $uri = $url->toString();
```

Comment:
Need to determine if this is acceptable

Reply:
Need to get URL from URI in module file in order to not have this changed

https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Url.php/function/Url%3A%3AfromUri/8.2.x